wagn/lib/wagn/generators/wagn/wagn_generator.rb

```diff
@@ -16,7 +16,7 @@ class WagnGenerator < Rails::Generators::AppBase
     desc: "Prepare deck for wagn core testing"
     
   class_option 'gem-path', :type => :string, aliases: '-g', :default => false, :group => :runtime, 
-    desc: "Path to local gem installation (Default, use env WAGN_DEV_GEM_PATH)"
+    desc: "Path to local gem installation (Default, use env WAGN_GEM_PATH)"
```

Comment:
Also shortened this constant name.

I set it up so my bash profile now has
````
export WAGN_GEM_PATH="~/dev/wagn/gem"
````
Loving this pattern!

Sorry if you've already done this and I broke it :(